Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM rustlang/rust:nightly-slim as builder
+
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        build-essential cmake libpq-dev libssl-dev libzmq3-dev pkg-config
+
+WORKDIR /srv/app
+
+COPY ffi ffi
+COPY src src
+COPY Cargo.lock Cargo.toml README.md ./
+
+RUN cargo build --release
+
+
+FROM debian:buster-slim
+
+ENV APP_DIR=/srv/app USER=rgbnode
+
+RUN adduser --home ${APP_DIR} --shell /bin/bash --disabled-login \
+        --gecos "${USER} user" ${USER}
+
+COPY --from=builder --chown=${USER}:${USER} \
+        ${APP_DIR}/target/release/ /usr/local/bin/
```

Comment:
Hm, are we copying content of /release/ dir here, not the dir itself?

Reply:
The release dir contains binary files, so I've put them in a standard linux directory for binaries. In this way you can access the container running the node and have `rgb-cli` available in PATH.

Yes, I've tried to run it and it seems working. These are the logs I see when running it:
```
[2020-09-15T15:24:16Z DEBUG rgb::contracts::fungible::processor] Instantiating RGB asset manager ...
[2020-09-15T15:24:16Z INFO  rgb::contracts::fungible::processor] RGB fungible assets schema file not found, creating one
[2020-09-15T15:24:16Z DEBUG rgb::contracts::fungible::cache::file] Instantiating RGB fungible assets storage (disk storage) ...
[2020-09-15T15:24:16Z DEBUG rgb::contracts::fungible::cache::file] RGB fungible assets data directory '"./data/testnet/cache/fungible"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::contracts::fungible::cache::file] Initializing assets file Some("./data/testnet/cache/fungible/assets.yaml") ...
[2020-09-15T15:24:16Z DEBUG rgb::contracts::fungible::runtime] Registering RGB20 schema
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] Instantiating RGB storage (disk storage) ...
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] RGB data directory '"./data/testnet/stash/default/"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] RGB schemata directory '"./data/testnet/stash/default/schemata"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] RGB geneses data directory '"./data/testnet/stash/default/geneses"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] RGB anchor data directory '"./data/testnet/stash/default/anchors"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::stash::storage::disk] RGB state transition data directory '"./data/testnet/stash/default/transitions"' is not found; creating one
[2020-09-15T15:24:16Z DEBUG rgb::stash::index::btree] Instantiating RGB index (file & memory storage) ...
[2020-09-15T15:24:16Z DEBUG rgb::stash::runtime] Received ZMQ RPC request: AddSchema(Schema { field_types: {0: String(8), 1: String(256), 2: String(65535), 3: Unsigned(Bit64, 0, 18446744073709551615), 4: Unsigned(Bit64, 0, 18446744073709551615), 5: Unsigned(Bit64, 0, 18446744073709551615), 6: Unsigned(Bit8, 0, 18), 7: Bytes(65535), 8: Integer(Bit64, 1593870844, 9223372036854775807)}, assignment_types: {0: StateSchema { format: Declarative, abi: {Validate: Standard(IssueControl)} }, 1: StateSchema { format: DiscreteFiniteField(Unsigned64bit), abi: {Validate: Standard(ConfidentialAmount)} }, 2: StateSchema { format: Declarative, abi: {Validate: Standard(Prunning)} }}, genesis: GenesisSchema { metadata: {0: Once, 1: Once, 2: NoneOrOnce, 3: Once, 4: Once, 5: NoneOrOnce, 6: Once, 8: Once}, defines: {0: NoneOrOnce, 1: NoneOrUpTo(None), 2: NoneOrUpTo(None)}, abi: {} }, transitions: {0: TransitionSchema { metadata: {4: Once}, closes: {0: Once}, defines: {0: NoneOrOnce, 1: NoneOrUpTo(None), 2: NoneOrUpTo(None)}, abi: {} }, 1: TransitionSchema { metadata: {}, closes: {1: OnceOrUpTo(None)}, defines: {1: NoneOrUpTo(None)}, abi: {} }, 2: TransitionSchema { metadata: {7: NoneOrUpTo(None)}, closes: {1: OnceOrUpTo(None), 2: OnceOrUpTo(None)}, defines: {1: NoneOrUpTo(None), 2: NoneOrUpTo(None)}, abi: {} }} })
[2020-09-15T15:24:16Z DEBUG rgb::stash::runtime] Got ADD_SCHEMA schema_data1pyqqqqqypqqqzqqyqqqsyqqylllsxqqqpqqqqqqqqqqqqq8llllllllllllsgqqqpqqqqqqqqqqqqq8lllllllllllls2qqqpqqqqqqqqqqqqq8llllllllllllsvqqqqyqpypcqqhll7zqqqyy0ezgqtuqqqqqqlllllllllllh7qcqqqqqqqgqqrlsyqgqqyqqsqqqqqqqqqqqqrllllllllllllcpqqq07qgzqqqqzqqqlupssqqqqqqsqqqqqqqqqqqqqyqqzqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqxqqpqqqqqqqqqqqqqpqqqyqqqqqqqqqqqqq9qqqqqqqqqqqqqqqqqcqqzqqqqqqqqqqqqqyqqqgqqqqqqqqqqqqqxqqqqqqqqqqqqqqqqqqqqyq0alllqqqqqqqqqqpqplhlluqqqqqqqqqqqqqqqqpsqqqqqyqqgqqpqqqqqqqqqqqqqqgqqqqqzqqqqqqqqqqqqqpsqqqqqqqqqqqqqqqqqqqpqrl0llcqqqqqqqqqqgq0alllqqqqqqqqqqqqqqqqqyqqqqqpqqqsplllluqqqqqqqqqqzqqpqrl0llcqqqqqqqqqqqqqqqqzqqqsqpcqlmll7qqqqqqqqqqzqqqsplllluqqqqqqqqqqyq8llllsqqqqqqqqqqsqqyq0alllqqqqqqqqqqpqplhlluqqqqqqqqqqqqqqqqqqqgty8vw
[2020-09-15T15:24:16Z DEBUG rgb::stash::runtime] API request processing complete
```

Reply:
Ok, b/c I thought `COPY  ${APP_DIR}/target/release/ /usr/local/bin/` will result in `/usr/local/bin/release/rgbd` and not `/usr/local/bin/rgbd`

Reply:
No, because the final `/` in the source directory instructs docker to copy the files that are inside the directory